yarn.lock

```diff
@@ -420,7 +432,7 @@ buffer-from@^1.0.0:
 
 buffer@^3.0.1:
   version "3.6.0"
-  resolved "https://registry.yarnpkg.com/buffer/-/buffer-3.6.0.tgz#a72c936f77b96bf52f5f7e7b467180628551defb"
+  resolved "http://registry.npmjs.org/buffer/-/buffer-3.6.0.tgz#a72c936f77b96bf52f5f7e7b467180628551defb"
```

Comment:
@vikaspotluri123 do you know why yarn would switch the links to the non-https npm registry as opposed to the https yarn registry? (maybe your `.npmrc` is pointed to that?)

Would prefer all of the links remain pointed to https if possible 😄

Reply:
I'm not sure what's going on, I cleared my cache and tried again but it still pulled in some http links. Since all of them pointed to NPM registry I manually updated them with sed 😕